entaxy-project/entaxy
# [TWG703B-99] Account currency conversion

### Summary

Dashboard doesn't take currency into account when making calculations. All numbers should be converted in the dashboard to the selected currency

### How to test it

Create accounts with different currencies as verify that the exchange rate is imported and dispayed correctly on the dashboard
